web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -324,119 +314,182 @@ Migrating your existing app to the new auth system is a two-step process:
 1. Run `wasp db migrate-dev` again to remove the redundant fields from the database.
-1. You can now delete the migration script and the `db.seeds` config.
+1. You can now delete the data migration function(s) you implemented earlier (e.g. in `src/migrateToNewAuth.ts`) and also the corresponding entries in the `app.db.seeds` field in `main.wasp` file.
 
 :::info Migrating a deployed app
 
   After doing the steps above successfully locally and making sure everything is working, it is time to push these changes to the deployed app again.
-  
-  _Deploy the app again_, either via `wasp deploy` or manually. Check our [Deployment docs](../advanced/deployment/overview.md) for more details. 
-  
-  The database migrations will automatically run on successful deployment of the server and delete the now redundant auth-related `User` columns from the database. 
-  
-  Your app is now fully migrated to the new auth system.
+
+  _Deploy the app again_, either via `wasp deploy` or manually. Check our [Deployment docs](../advanced/deployment/overview.md) for more details.
+
+  The database migrations will automatically run on successful deployment of the server and delete the now redundant auth-related `User` columns from the database.
+
+  Your app is now fully migrated to the new auth system 🎉!
 
 :::
 
+### Next steps
 
-#### Example Migration Functions
+If you made it this far, you've completed all the necessary steps to get your
+Wasp app working with Wasp 0.12.x. Nice work!
+
+Finally, since Wasp no longer requires you to separate your client source files
+(previously in `src/client`) from server source files (previously in
+`src/server`), you are now free to reorganize your project however you think is best,
+as long as you keep all the source files in the `src/` directory.
+
+This section is optional, but if you didn't like the server/client
+separation, now's the perfect time to change it!
+
+For example, if your `src` dir looked like this:
+
+```
+src
+│
+├── client
+│   ├── Dashboard.tsx
+│   ├── Login.tsx
+│   ├── MainPage.tsx
+│   ├── Register.tsx
+│   ├── Task.css
+│   ├── TaskLisk.tsx
+│   ├── Task.tsx
+│   └── User.tsx
+├── server
+│   ├── taskActions.ts
+│   ├── taskQueries.ts
+│   ├── userActions.ts
+│   └── userQueries.ts
+└── shared
+    └── utils.ts
+```
+
+you can now change it to a feature-based structure (which we recommend for any project that is not very small):
+
+```
+src
+│
+├── task
+│   ├── actions.ts    -- former taskActions.ts
+│   ├── queries.ts    -- former taskQueries.ts
+│   ├── Task.css
+│   ├── TaskLisk.tsx
+│   └── Task.tsx
+├── user
+│   ├── actions.ts    -- former userActions.ts
+│   ├── Dashboard.tsx
+│   ├── Login.tsx
+│   ├── queries.ts    -- former userQueries.ts
+│   ├── Register.tsx
+│   └── User.tsx
+├── MainPage.tsx
+└── utils.ts
+```
+
+
+## Addendum
```

Comment:
This is new -> so what I did is added `Addendum` chapter at the top level, and put `Example Data Migration Functions` down there.
This way, once you are done with the Auth migration, you don't hit the `Example Data Migration Functions` again and then have to scroll through all of them till you find Next Steps, instead you are immediately at the Next Steps.
Also, this makes it clear that this chapter is not part of the normal flow but is an additional information on this page.

Reply:
I'd advise against using a complex word like `Addendum` and call the section something like `Extras` or `Additional info`

Reply:
Sounds good, I changed it to Appendix!

Reply:
Appendix is better than Addendum but it still feels a bit as an advanced vocabulary work to me 😄 but no need to change